demos/JWST/S1_miri_template.ecf

```diff
@@ -26,6 +26,16 @@ skip_gain_scale     False
 #Pipeline stages parameters
 jump_rejection_threshold  4.0 #float, default is 4.0, CR sigma rejection threshold
 
+#Background subtraction
```

Comment:
Please document all new ECF/EPF settings in docs/source/ecf.rst and also update the ECF/EPF files in docs/media/